WordPress/Classes/ViewRelated/Gutenberg/GutenbergWeb/GutenbergWebViewController.swift

```diff
@@ -100,6 +106,16 @@ class GutenbergWebViewController: GutenbergWebSingleBlockViewController, WebKitA
         }
     }
 
+    private func loadCustomStyles(named name: String, with argument: String? = nil) -> WKUserScript? {
```

Comment:
I've [proposed an idea on the Gutenberg PR](https://github.com/WordPress/gutenberg/pull/37221#discussion_r771093813) about how we may be able to simplify this a little.

Reply:
If we keep this function the way it is we may want to drop `with argument: String? = nil` since it isn't currently being used.

Reply:
I've gone ahead to update this as per your suggestion in https://github.com/wordpress-mobile/WordPress-iOS/pull/17644/commits/df2615cd5bd97a55fd44612dcf6548a7a32dda2e :)